pypika/dialects.py

```diff
@@ -781,6 +781,16 @@ def drop_view(self, view: str) -> "ClickHouseDropQueryBuilder":
 class ClickHouseQueryBuilder(QueryBuilder):
     QUERY_CLS = ClickHouseQuery
 
+    def __init__(self, **kwargs) -> None:
+        super().__init__(self, as_keyword=True)
```

Comment:
is there no need to send other `kwargs` to the `super().__init__`?

Reply:
I think my previous code was indeed wrong, it has been corrected on the latest revision, thanks for bringing up the issue